### ISSUE TITLE
Breakpoints are broken if the full path contains parentheses.

### ISSUE THREAD

**Issue by fushihara** [TypeStrong/ts-node#1905]
If the full path of the project contains parentheses, breakpoint behavior will be broken.
I don't believe this at all, but I reproduced it on a new installation of nodejs and vscode on a windows PC that I don't use regularly.

If I press F5 to run debug with the first screenshot, the breakpoint stops at a different ts file, as shown in the second screenshot.
The full path of the project at this time is a path like `c:\Users\xxxxxx\(a)\`

If I change the full path of the project to `c:\Users\xxxxxx\a\`, the breakpoint works on the open ts file as shown in the third screenshot. I think this is the normal state.

The same phenomenon occurred when the full path of the project included in the `[a-b]` format.

The windows version is 19044.1682 node.js version is 18.12.0
The contents of the other files are as follows

![image](https://user-images.githubusercontent.com/1039534/198842945-caa1955d-28c1-4ba7-a1fb-04bf85834380.png)
![image](https://user-images.githubusercontent.com/1039534/198842955-79dfc6c3-9c0e-4fa0-a74a-2895d7691159.png)
![image](https://user-images.githubusercontent.com/1039534/198842963-a132cb8f-4e80-46e3-b667-17d3ca0e2949.png)

### Search Terms
ts-node vscode breakpoint not working js

### Minimal reproduction
`/vscode/launch.json`
```json:.
{
  "version": "0.2.0",
  "configurations": [
    {
      "name": "index.ts",
      "type": "node",
      "request": "launch",
      "runtimeArgs": ["--nolazy", "-r", "ts-node/register"],
      "args": ["${workspaceFolder}/src/index.ts"]
    }
  ]
}
```

`/src/index.ts`
```ts
debugger;
```

`/package.json`
```
{
  "dependencies": {
    "ts-node": "10.9.1",
    "typescript": "4.8.4"
  }
}
```

**Comment by cspotcode** [comment]
Based on my investigation, this is not a ts-node problem.  I did an experiment where I removed ts-node and the problem still occurred: I compiled with `tsc` and then ran the debugger.  The same problem happened, *without* using ts-node.  This makes me believe that this is a problem with VSCode, with node, or with tsc.

Here is my reproduction.  It is configured with two debug configurations: one that uses ts-node, and the other that does not.
https://github.com/TypeStrong/ts-node-repros/tree/issue-1905

Are you able to see the same as me?  That this problem occurs without ts-node?
